src/main/java/com/scalar/db/storage/jdbc/query/InsertQuery.java

```diff
@@ -0,0 +1,93 @@
+package com.scalar.db.storage.jdbc.query;
+
+import com.scalar.db.io.Key;
+import com.scalar.db.io.Value;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class InsertQuery extends AbstractQuery {
+
+  public static class Builder {
+    private final String fullTableName;
+    private Key partitionKey;
+    private Optional<Key> clusteringKey;
+    private Map<String, Value> values;
+
+    Builder(String fullTableName) {
```

Comment:
Just a question. 
Some `XXXQuery` accepts `fullTableName` in its builder's constructor, but some accept it with its builder's method (like SelectQuery).
Do you have any particular reason for that?

Reply:
The reason is that only `SelectQuery` accepts projections as a first parameter while other `XXXQuery` accepts a full table name as a first parameter. 

For example:
```
// SelectQuery
queryBuilder.select(<projections>)
  .from(<the full table name>)
  .where(...)
  .build()

// InsertQuery
queryBuilder.insertInto(<a full table name>)
  .values(...)
  .build()

// UpdateQuery
queryBuilder.update(<a full table name>)
  .set(...)
  .where(...)
  .build()

// DeleteQuery
queryBuilder.deleteFrom(<a full table name>)
  .where(...)
  .build()

// UpsertQuery
queryBuilder.upsertInto(<a full table name>)
  .values(...)
  .build()
```

As you can see, only `SelectQuery` doesn't accept a full table name as a first parameter, but the others accept a full table name as a fist parameter.